PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -1,3 +1,70 @@
+$GraphEndpoints = @{
+    "Get-MgBetaIdentityConditionalAccessPolicy" = "https://graph.microsoft.com/beta/identity/conditionalAccess/policies"
+    "Get-MgBetaSubscribedSku" = "https://graph.microsoft.com/beta/subscribedSkus"
+    "Get-MgBetaDirectoryRole" = "https://graph.microsoft.com/beta/directoryRoles"
+    "Get-MgBetaDirectoryRoleMembers" = "https://graph.microsoft.com/beta/directoryRoles/{0}/members"
+    "Get-MgBetaRoleManagementDirectoryRoleEligibilityScheduleInstance" = "https://graph.microsoft.com/beta/roleManagement/directory/roleEligibilityScheduleInstances"
+    "Get-MgBetaRoleManagementDirectoryRoleAssignmentScheduleInstance" = "https://graph.microsoft.com/beta/roleManagement/directory/roleAssignmentScheduleInstances"
+    "Get-MgBetaIdentityGovernancePrivilegedAccessGroupEligibilityScheduleInstance" = "https://graph.microsoft.com/beta/identityGovernance/privilegedAccess/group/eligibilityScheduleInstances"
+    "Get-MgBetaPrivilegedAccessResource" = "https://graph.microsoft.com/beta/privilegedAccess/aadGroups/resources"
+    "Get-MgBetaPolicyAuthorizationPolicy" = "https://graph.microsoft.com/beta/policies/authorizationPolicy"
+    "Get-MgBetaDirectorySetting" = "https://graph.microsoft.com/beta/settings"
+    "Get-MgBetaPolicyAuthenticationMethodPolicy" = "https://graph.microsoft.com/beta/policies/authenticationMethodsPolicy"
+    "Get-MgBetaDomain" = "https://graph.microsoft.com/beta/domains"
+    "Get-MgBetaOrganization" = "https://graph.microsoft.com/beta/organization"
+    "Get-MgBetaUser" = "https://graph.microsoft.com/beta/users"
+    "Get-MgBetaPolicyRoleManagementPolicyAssignment"  = "https://graph.microsoft.com/beta/policies/roleManagementPolicyAssignments"
+}
+
+function Invoke-GraphDirectly {
```

Comment:
Can you please add a test case for this function that mocks out Invoke-MgGraphRequest?

Reply:
This comment refers to a Powershell unit test. Touch base with @schrolla with specifics on how to do that.

Reply:
Added a basic test to cover er this function